sentry-android-core/src/test/java/io/sentry/android/core/ScreenshotEventProcessorTest.kt

```diff
@@ -153,5 +155,25 @@ class ScreenshotEventProcessorTest {
         assertNull(hint.screenshot)
     }
 
+    @Test
+    fun `when enabled, the feature is added to the integration list`() {
+        SentryIntegrationPackageStorage.getInstance().clearStorage()
+        val hint = Hint()
+        val sut = fixture.getSut(true)
+        val event = fixture.mainProcessor.process(getEvent(), hint)
+        sut.process(event, hint)
+        assertTrue(fixture.options.sdkVersion!!.integrationSet.contains("ScreenshotEventProcessor"))
```

Comment:
Can we also drop the `EventProcessor` part in `getIntegrationName` so they become `Screenshot` and `ViewHierarchy` essentially?

Reply:
Sure thing.